docs/Custom-SideChannels.md

```diff
@@ -193,7 +193,7 @@ messages to the Unity environment from Python using it.
 string_log = StringLogChannel()
 
 # We start the communication with the Unity Editor and pass the string_log side channel as input
-env = UnityEnvironment(base_port=UnityEnvironment.DEFAULT_EDITOR_PORT, side_channels=[string_log])
+env = UnityEnvironment(side_channels=[string_log])
```

Comment:
Since these should now do the correct thing, I removed the explicit port (for simplicity and future-proofing) wherever I found it in the docs.